velox/functions/lib/window/tests/WindowTestBase.h

```diff
@@ -175,5 +173,9 @@ class WindowTestBase : public exec::test::OperatorTestBase {
       const std::string& overClause,
       const std::string& frameClause,
       const std::string& errorMessage);
+
+  /// Function arguments parse options to tell the DuckDB parser whether to parse
```

Comment:
Nit : Reword "ParseOptions for the DuckDB Parser. nth_value in Spark expects to parse integer as bigint vs bigint in Presto. The default is to parse integer as bigint (Presto behavior)."

Reply:
Thanks, updated.